### PR TITLE
added sharpen function and its test

### PR DESCRIPTION
**Pull request by chaomander2018**
Hi team, 

I have added my sharpen function and its test script to the ml2_sharpen branch. Please review my code and let me know if you have any questions. Thank you!

### CODE REVIEW

File: filterizePy/sharpen.py [hunk 1]
```diff
@@ -4,36 +4,95 @@
...
+    index = input_path.rfind("/") + 1
+    output_path = input_path[:index] + "sharpened_" + input_path[index:]
+
 
-    # Apply sharpen filter to the image, this function will be complete by milestone 2.
-    # output_img = input_img, i will need to finish this next week
-    skimage.io.imsave("../img/sharpened_img.png", output_img)
+    skimage.io.imsave(output_path, output_img)
+    #output_path = path+output_name
+    return output_path
```
Comment (jackattackyang) [inline filterizePy/sharpen.py]: I like that the output file path is returned. I should add that as well

File: filterizePy/test/test_sharpen.py [hunk 2]
```diff
@@ -5,38 +5,52 @@
...
 
-def check_input_type(input_img):
+def test_input_type():
     """
     This function checks the input image format is one of the valid image formats.
-
     """
-    assert imghdr.what(input_img) in ['png','jpeg','gif','bmp','jpg'] , "The input image format is incorrect, try another format"
-
+    assert imghdr.what("img/mirror.png") == 'png', "Not a png file, please try again"
```
Comment (jackattackyang) [inline filterizePy/test/test_sharpen.py]: We can try reading in other image types as well!

**Comment by akanshaVashisth** [comment]
Hi Chao,

All looks good. You could add a test to compare the initial image with the filtered one as discussed.
Good job over all!

File: filterizePy/sharpen.py (hunk 1 above)
Comment (chaomander2018) [reply filterizePy/sharpen.py]: Thank you Jack!

File: filterizePy/test/test_sharpen.py (hunk 2 above)
Comment (chaomander2018) [reply filterizePy/test/test_sharpen.py]: Ok, will revert the change, thank you!

File: filterizePy/sharpen.py [hunk 3]
```diff
@@ -4,36 +4,95 @@
 # You may obtain a copy of the License at https://mit-license.org
 
 # This script sharpens an image.
-# Input  : A path to an image in .png, .jpeg,.gif,.bmp, .jpg format
-# Output : A path to a sharpened image in the same format as the input image file type
+# Input  : A path to an image in .png format
+# Output : A path to a sharpened .png image
```
[the remark this answers is not in the document]
Comment (chaomander2018) [reply filterizePy/sharpen.py]: Will do!

**Comment by chaomander2018** [comment]
@akanshaVashisth, thank you for your feedback, I will add one more test!

**Pull request merged**